datafusion/sqllogictest/test_files/array.slt

```diff
@@ -2788,7 +2798,6 @@ select array_append(column1, arrow_cast(make_array(1, 11, 111), 'FixedSizeList(3
 
 # DuckDB: [4]
 # ClickHouse: Null
-# Since they dont have the same result, we just follow Postgres, return error
```

Comment:
Removed this since the query doesn't seem to error (am I misunderstanding what the comment means 🤔 )